### PR TITLE
LTD-5659 Add simple health check

### PR DESCRIPTION
**Pull request by markj0hnst0n**
### Aim

DBT platform team needs to hit an endpoint and get a 200 to make sure that the django app is up and running

[LTD-5659](https://uktrade.atlassian.net/browse/LTD-5659)

[LTD-5659]: https://uktrade.atlassian.net/browse/LTD-5659?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: unit_tests/core/test_healthcheck.py [hunk 1]
```diff
@@ -66,3 +67,21 @@ def test_healthcheck_down(backends):
         b"</status><response_time>0.23</response_time>"
         b"</pingdom_http_custom_check>\n"
     )
+
+
+def test_service_available_check_broken(client, backends):
+    backends.reset()
+    backends.register(HealthCheckBroken)
```
Comment (depsiatwal) [inline unit_tests/core/test_healthcheck.py:74]: why we registering the HealthCheckBroken then asserting everything is  OK

Comment (markj0hnst0n) [reply unit_tests/core/test_healthcheck.py:74]: To ensure that even if one of the health checks breaks we still get a 200 response form the endpoint

Comment (depsiatwal) [reply unit_tests/core/test_healthcheck.py:74]: ok that makes sense may be a comment to clarify why the test expects a 200

Comment (markj0hnst0n) [reply unit_tests/core/test_healthcheck.py:74]: Done!

File: core/health_check/views.py [hunk 2]
```diff
@@ -8,3 +10,11 @@ def render_to_response(self, context, status):
...
         context["total_response_time"] = sum([plugin.time_taken for plugin in context["plugins"]])
         return super().render_to_response(context=context, status=status, content_type="text/xml")
+
+
+class ServiceAvailableHealthCheckView(MainView):
+    def get(self, request, *args, **kwargs):
+        return self.render_to_response()
+
+    def render_to_response(self):
+        return HttpResponse(status.HTTP_200_OK)
```
Comment (saruniitr) [inline core/health_check/views.py:20]: Is it worth checking to see if we can access db also or it doesn't make it simple anymore?

Comment (markj0hnst0n) [reply core/health_check/views.py:20]: Kevin had asked specifically that we don't do a db check and that we just check that the web service for the application is up

**Review by saruniitr — APPROVED** [review]

**Pull request merged**